Example/MBViewController.m

```diff
@@ -114,7 +114,9 @@ - (void)startNavigation:(MBRoute *)route {
     MBNavigationService *service = [[MBNavigationService alloc] initWithRoute:route directions:self.directions locationSource:nil eventsManagerType:nil simulating:MBNavigationSimulationOptionsAlways routerType:nil];
     
     self.navigation = service;
-    MBNavigationViewController *controller = [[MBNavigationViewController alloc] initWithRoute:route styles:nil navigationService:service voiceController: nil];
+    MBNavigationOptions *options = [[MBNavigationOptions alloc] init];
+    options.navigationService = service;
```

Comment:
Setting just the `navigationService` property seems like a common enough scenario that maybe we should provide a dedicated initializer for it.